email_notification/helpers/email_translations.py

```diff
@@ -0,0 +1,320 @@
+        "sv": "untilApplicationPeriodEnd SV",
+    },
+    #
+    # RESERVATION_CANCELLED
+    "subjectReservationCancelled": {
+        "fi": "Varauksesi on peruttu",
+        "en": "Your booking has been cancelled",
+        "sv": "Din bokning har avbokats",
+    },
+    "textReservationCancelled": {
+        "fi": "Varauksesi on peruttu.",
+        "en": "Your booking has been cancelled.",
+        "sv": "Din bokning har avbokats.",
+    },
+    "cancellationReason": {
+        "fi": "Peruutuksen syy",
+        "en": "Your reason for cancellation",
+        "sv": "Din anledning till avbokning",
+    },
+    "cancellationInstructions": {
+        "fi": "Lisätietoa peruutuksesta",
+        "en": "Additional information about cancellation",
+        "sv": "Mer information om avbokning",
+    },
+    #
+    # RESERVATION_CONFIRMED
+    "subjectReservationConfirmed": {
+        "fi": "Varauksesi on vahvistettu",
+        "en": "Thank you for your booking at Varaamo",
+        "sv": "Din bokning är bekräftad",
+    },
+    "textReservationConfirmed": {
+        "fi": "Olet tehnyt uuden varauksen:",
+        "en": "You have made a new booking:",
+        "sv": "Du har gjort en ny bokning:",
+    },
+    #
+    # RESERVATION_HANDLED_AND_CONFIRMED
+    "subjectReservationHandledConfirmed": {
+        "fi": "Varauksesi on vahvistettu",
+        "en": "Your booking is confirmed",
+        "sv": "Din bokning är bekräftad",
+    },
+    "textReservationHandledConfirmed": {
+        "fi": "Varauksesi on nyt vahvistettu.",
+        "en": "Your booking is now confirmed.",
+        "sv": "Din bokning har bekräftats.",
+    },
+    "textReservationHandledConfirmedSubsidized": {
+        "fi": "Varauksesi on hyväksytty, ja varaukseen on myönnetty seuraava alennus:",
+        "en": "Your booking has been confirmed with the following discount:",
+        "sv": "Din bokning har bekräftats med följande rabatt:",
+    },
+    #
+    # RESERVATION_HANDLING_REQUIRED
+    "subjectReservationHandlingRequired": {
+        "fi": "Varauksesi odottaa käsittelyä",
+        "en": "Your booking is waiting for processing",
+        "sv": "Din bokning väntar på att behandlas",
+    },
+    "textReservationHandlingRequired": {
+        "fi": "Olet tehnyt alustavan varauksen:",
+        "en": "You have made a new booking request:",
+        "sv": "Du har gjort en ny bokningsförfrågan:",
+    },
+    "pendingNotification": {
+        "fi": "Saat varausvahvistuksen sähköpostitse, kun varauksesi on käsitelty. Otamme sinuun yhteyttä, jos tarvitsemme lisätietoja varauspyyntöösi liittyen.",
+        "en": "You will receive a confirmation email once your booking has been processed. We will contact you if further information is needed regarding your booking request.",
+        "sv": "Du kommer att få en bekräftelse via e-post när din bokning har behandlats. Vi kommer att kontakta dig om ytterligare information behövs angående din bokningsförfrågan.",
+    },
```

Comment:
Wrap long strings in parenthesis so that they fit on one line.

```python
{
    "fi": (
        "Saat varausvahvistuksen sähköpostitse, kun varauksesi on käsitelty. "
        "Otamme sinuun yhteyttä, jos tarvitsemme lisätietoja varauspyyntöösi liittyen.",
    )
}
```

Reply:
I intentionally kept then all in a single line for readability, but will change 👍